manager/manager/settings.py

```diff
@@ -17,6 +17,10 @@
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
 
+# Define the URL subpath for this application when deployed:
+FORCE_SCRIPT_NAME = os.environ.get("URL_SUBPATH")
```

Comment:
You could set the default value here as `""` so that the `if FORCE_SCRIPT_NAME` checks become unnecessary.

Reply:
I wanted to do that in a beginning, but then I realized this is a Django setting variable, thus it is thought to be `None` if no string is defined. I did't try this out, but I think if set `""` as default value, some things could stop working.